uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsencePeriod.php

```diff
@@ -258,6 +258,25 @@ public static function getMostRecentStartDateAvailable()
   }
 
   /**
+   * Returns the current AbsencePeriod. That is, the period that contains the
+   * current date. If no such period is found, null will be returned.
+   *
+   * @return \CRM_HRLeaveAndAbsences_BAO_AbsencePeriod|null
+   */
+  public static function getCurrentPeriod()
+  {
+    $period = new self();
+    $period->whereAdd("start_date <= NOW()");
```

Comment:
No need. In this situation NOW() and CURDATE() will work pretty much the same way. The tests are still passing.